cmd/zq/README.md

```diff
@@ -0,0 +1,102 @@
+# `zq`
+
+`zq` is a command-line tool to search, analyze, and transform structured logs. 
+ It evaluates [ZQL ](../../zql/docs/README.md) queries against input log
+  files, producing an output log stream in the [ZNG](../../zng/docs/spec.md)
+  format by default. For all `zq` options, see the built-in help by running:
+
+```
+zq help
+```
+
+## Examples
+
+Here are a few examples using a small Zeek formatted log file, `conn.log
+`, located in this directory. See the
+[zq-sample-data repo](https://github.com/brimsec/zq-sample-data) for more test
+data, which is used in the examples in the
+[query language documentation](../../zql/docs/README.md).
+
+To cut the columns of a Zeek "conn" log like `zeek-cut`, and output to the
+ terminal run:
+
+```
+zq -t "* | cut ts,id.orig_h,id.orig_p" conn.log
+```
+
+The `-t` tells `zq` to use [TZNG](../../zng/docs/spec.md#4-zng-text-format-tzng)
+for its output format. The "`*`" 
+tells `zq` to match every line, which is sent to the `cut` processor
+using the UNIX-like pipe syntax.
+
+When looking over everything like this, you can omit the search pattern
+as a shorthand and simply type:
+```
+zq -t "cut ts,id.orig_h,id.orig_p" conn.log
+```
+
+The default output is the binary ZNG format.  If you want just the tab-separated
+ lines like `zeek-cut`, you can specify text output:
+```
+zq -f text "cut ts,id.orig_h,id.orig_p" conn.log
+```
+If you want the old-style Zeek [ASCII TSV](https://docs.zeek.org/en/stable/examples/logs/)
+log format, run the command with the `-f` flag specifying `zeek` for the output
+format:
+```
+zq -f zeek "cut ts,id.orig_h,id.orig_p" conn.log
+```
+You can use an aggregate function to summarize data over one or
+more fields, e.g., summing field values, counting, or computing an average.
+```
+zq -t "sum(orig_bytes)" conn.log
+zq -t "orig_bytes > 10000 | count()" conn.log
+zq -t "avg(orig_bytes)" conn.log
+```
+
+The [ZNG specification](../../zng/docs/spec.md) describes the significance of
+ the
+`_path` field.  By leveraging this, diverse Zeek logs can be combined into a single
+file.
```

Comment:
```suggestion
The [ZNG specification](../../zng/docs/spec.md) describes how the format can
represent a stream of heterogeneously typed records. By leveraging this,
diverse Zeek logs can be combined into a single file.
```

This was content that you didn't change, but during this review pass I realize it's gone stale. I think this suggestion brings it current.